### ISSUE TITLE
Use Puppeteer

### ISSUE THREAD

**Issue by STRML** [oblador/loki#31]
Adding https://github.com/GoogleChrome/puppeteer as a target (easily installable via npm) should make this work out-of-the-box in most dev and CI platforms.

**Comment by oblador** [comment]
Hi, the idea is to use docker to run chrome which is deterministic and will run the same on all platforms/ci environments. What would be the advantage of using puppeteer if you’re already running chrome in docker?

**Comment by STRML** [comment]
At the very least the chrome.app target could be dropped. There could then be a local target for linux/windows/osx, not just osx. It could also obviate the need for docker, as Puppeteer should be a completely clean environment. This would reduce system dependencies and make it a little easier to get started.

**Comment by oblador** [comment]
Yeah, that's a fair point. I guess I'm a bit hesitant to force 100MB down peoples throat even if they don't use it. But I'll put it on my todo!

**Comment by STRML** [comment]
Could be useful as an option: e.g. if `require('puppeteer')` doesn't throw, allow a configuration type of `chrome.puppeteer`.

**Comment by penx** [comment]
@oblador any pointers as to where this should be implemented? I may be able to do it
